index.js

```diff
@@ -689,6 +696,47 @@ class Hyperbee extends ReadyResource {
     return (this._checkout <= this.core.length || this._checkout <= 1) ? this.core.snapshot() : this.core.session({ snapshot: false })
   }
 
+  async clearUnlinked (options = {}) {
+    await this.ready()
+
+    const { gte = 0, lt = this.version - 1 } = options
```

Comment:
Shouldn't this be `lt = this.version`? (or `lte = this.version - 1`)

Reply:
In that case we could add a
`if (lt >= this.version) throw new Error('you are doing something stupid')`, but that's maybe to defensive

Reply:
I like that, add to pr